example/src/Views/Button.tsx

```diff
@@ -27,13 +27,11 @@ export default class TestButton extends React.Component {
     Alert.alert(text);
   };
 
-  render(): React.ReactNode {
+  render() {
     const { disabled, not100 } = this.state;
-    const itemStyle = styleReferenceBreaker(styles.baseSpacing);
+    const itemStyle: ViewStyle = styleReferenceBreaker(styles.baseSpacing);
```

Comment:
adding the type hint still checks that the assignment makes sense + allows addition of other styles